src/rtc_time.cpp

```diff
@@ -169,4 +170,18 @@ bool RTCTime::operator>=(const RTCTime &rhs) const {
 long long RTCTime::operator-(const RTCTime &rhs) const {
     return (getTotalSeconds() - rhs.getTotalSeconds());
 }
+
+RTCTime &RTCTime::operator-=(const RTCTime &rhs) {
+    this->setTotalSeconds(*this - rhs);
+    return *this;
+}
+
+long long RTCTime::operator+(const RTCTime &rhs) const {
```

Comment:
This should return an RCTTime struct?
If thats not the case, make it an unsigned long to avoid type conversion